7_deploy_osh/play.yml

```diff
@@ -29,11 +29,11 @@
         - developer_mode
         - build_osh_images | default('False') | bool
 
-- hosts: osh-deploy-cp
+- hosts: osh-deploy-workers
   gather_facts: yes
   any_errors_fatal: true
   roles:
-    - role: suse-osh-setup
+    - name: suse-osh-setup-workers
```

Comment:
Are you sure the `name` is correct?

Reply:
Good question. Anyway, everytthing else in this playbook uses `role` so I think it's cleaner to stay with `role`